src/components/Form.js

```diff
@@ -129,6 +129,12 @@ const Label = styled.label``;
 const Input = styled.input`
   border-radius: 6px;
   padding-bottom: 0px;
+  border: 2px solid var(--passive);
+  cursor: pointer;
+  &:focus {
+    outline: none;
+    border: 2.5px solid var(--active-main);
+  }
```

Comment:
Also remember the on prefix for the handleCreateCreepypasta and onCreepypastaEdit.